ldapjs/index.js

```diff
@@ -0,0 +1,77 @@
+const ldap = require('ldapjs');
+const fs = require('fs');
+const path = require('path');
+
+const server = ldap.createServer();
+
+server.bind('cn=root', function(req, res, next) {
```

Comment:
Would it be better to have this configurable rather than a string here?

Reply:
Yes, I will change this.